### ISSUE TITLE
Support  Cron Trait

### ISSUE THREAD

**Issue by wonderflow** [crossplane/addon-oam-kubernetes-local#13]
We used to discuss it in Rudr https://github.com/oam-dev/rudr/issues/40
I think this is really a common trait which can be `core` trait candidate

**Comment by allenhaozi** [comment]
Cron Trait is a very general requirement
we tried to develop a cron trait internally
If it works and appropriate, we can contribute it

**Comment by wonderflow** [comment]
Really cool, willing to assign this issue to you.

**Comment by hasheddan** [comment]
@wonderflow @allenhaozi I think this will likely go in https://github.com/crossplane/crossplane

**Comment by resouer** [comment]
Yes, let's move the discussion to Crossplane core repo, it's important application layer feature. /cc @wonderflow 

Though I'm not sure it's general enough as a core trait, standard trait feels more accurate.

**Comment by wonderflow** [comment]
yeah, let me create an issue in crossplane core repo.
